From 93cce1e4f9df424a111a02ab53bc9c772a33fa59 Mon Sep 17 00:00:00 2001
From: Paul Barrette <paul.barrette@windriver.com>
Date: Fri, 18 May 2012 14:14:46 -0400
Subject: [PATCH] lttng: force lttng Kconfig to be modular only

The default lttng-modules tree is meant to be compiled as an external kernel
module and does not have a Kconfig.  The LTSI kernel added lttng 2.0 to their
kernel tree and this Kconfig can be seen at:
http://git.linuxfoundation.org/?p=ltsi-kernel.git;a=blob;f=patches.lttng/0017-lttng-toplevel-Makefile-and-Kconfig.patch

The changes made from the above:
       -enforce compiling as a kernel module

Note: this driver must be compiled as a kernel module.  The lttng-tools
expects to modprobe all the lttng modules and fails otherwise.

Signed-off-by: Paul Barrette <paul.barrette@windriver.com>

diff --git a/drivers/staging/lttng/Kconfig b/drivers/staging/lttng/Kconfig
index 34c4a4f..7eff7cf 100644
--- a/drivers/staging/lttng/Kconfig
+++ b/drivers/staging/lttng/Kconfig
@@ -1,6 +1,6 @@
 config LTTNG
 	tristate "LTTng kernel tracer"
-	depends on TRACEPOINTS
+	depends on TRACEPOINTS && m
 	help
 	  The LTTng 2.0 Tracer Toolchain allows integrated kernel and
 	  user-space tracing from a single user interface: the "lttng"
-- 
1.8.3.1

